llama_cloud_services/parse/base.py

```diff
@@ -1198,10 +1194,11 @@ async def _aget_json(
     ) -> List[dict]:
         """Load data from the input path."""
         try:
-            job_id = await self._create_job(file_path, extra_info=extra_info)
-            if self.verbose:
-                print("Started parsing the file under job_id %s" % job_id)
-            result = await self._get_job_result(job_id, "json")
```

Comment:
Batching worked properly here before since all the `aget_json` based calls batched at the `aget_json()` level and not at the `_create_job()`/`_get_job_result()` levels